app/views/admin/events/index.csv.haml

```diff
@@ -0,0 +1,24 @@
+- headers = ['Title',
+             'Abstract',
+             'Start time',
+             'Submitter',
+             'Speaker',
+             'Event Type',
+             'Track',
+             'Difficulty Level',
+             'Room',
+             'State']
+= CSV.generate_line headers
+- @events.each do |event|
+  = CSV.generate_line([event.title, event.abstract, event.time,
+                       event.submitter.name, event.speaker_names, event.event_type.title,
+                       (event.track.present? ? event.track.name : 'Nil'),
```

Comment:
In the other files we have '' instead of 'Nil'.  @ChrisBr do we prefer 'nil' here in CSV? (Instead of showing nothing)